docs/source/gbd2017_models/risk_attributable_causes/vitamin_a_deficiency/index.rst

```diff
@@ -54,15 +95,90 @@ Below is a list of measures and corresponding IDs:
 	  - gbd_id = reiid(96)
 	  - como, use get_measure
 
+Risk Factor Hierarchy
+^^^^^^^^^^^^^^^^^^^^^
+
+.. image:: vitA_risk_hierarchy.svg
+
+Restrictions
+^^^^^^^^^^^^
+
+.. list-table:: GBD 2017 Risk Restrictions
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     -
+     -
+   * - Female only
+     -
+     -
+   * - YLL only
+     -
+     -
+   * - YLD only
+     -
+     -
+   * - YLL age group start
+     - Post Neonatal
+     - (28, 364 days], age_group_id = 4
+   * - YLL age group end
+     - 1-4 years
+     - (1, 4 years], age_group_id = 5
+   * - YLD age group start
+     - Early Neonatal
+     - (0, 6 days], age_group_id = 2
+   * - YLD age group end
+     - 95 Plus
+     - (95, 125], age_group_id = 235
```

Comment:
Again, the age group intervals should be closed on the left and open on the right, with the right endpoint being the next starting age: [28, 365 days), [1, 5 years), [0, 7 days), [95, 125 years).

Reply:
got it, thanks

Reply:
The age group _names_, on the other hand, do match what you have.  E.g. "1 - 4 years" is the correct age group name for id=5.